scale/src/commonMain/kotlin/Ticks.kt

```diff
@@ -0,0 +1,89 @@
+package com.juul.krayon.scale
+
+import kotlin.math.abs
+import kotlin.math.floor
+import kotlin.math.log10
+import kotlin.math.pow
+import kotlin.math.roundToInt
+import kotlin.math.sqrt
+
+// A friendly reminder to thank your local math-magician, because I don't know what's happening in most of this file.
```

Comment:
Yeah the above stuff is difficult to follow along but I understand enough.. but whats going on here 🤷‍♂️